api/src/main/java/org/hawkular/apm/api/model/trace/Node.java

```diff
@@ -324,16 +348,18 @@ public void addInteractionId(String id) {
      * @param scope The scope
      * @return The subset of correlation ids that are associated with the scope
      */
-    public List<CorrelationIdentifier> getCorrelationIds(Scope scope) {
+    public List<CorrelationIdentifier> findCorrelationIds(Scope... scope) {
         List<CorrelationIdentifier> ret = null;
 
         for (int i=0; i < correlationIds.size(); i++) {
```

Comment:
Where code is being used in the processing of the events, rather than in support of the UI queries, I prefer to go for performance over readability.